### PR TITLE
Add Nemo - Nemo is your digital place for focus.

### PR DESCRIPTION
**Pull request by harshitsinghai77**
Add Nemo under `Apps & Examples` section.

## Describe Why This Is Awesome
Nemo is an open-source side project app focusing on productivity, focus, and getting things done. Nemo analytics help you get insights on how you've spent your day and how productive have you been in the week.

Nemo is 100% free, open-source, and provides advanced analytics on your productivity. You can define the tasks you're working on. It also has a Pomodoro timer. You can customize all the settings and create your perfect environment. You can tune out other noises by listening to calm and relaxing nature sounds. It also records the tasks you're working on and performs analytics to give you a better view of all the tasks you've worked on.

Nemo uses Deta Micros for deployment.

## Demo
Try [Nemo](https://nemo-landing-page.netlify.app/)
Project: [Github](https://github.com/harshitsinghai77/nemo-backend)

## Project Links
Backend: https://github.com/harshitsinghai77/nemo-backend
Frontend: https://github.com/harshitsinghai77/nemo-frontend

## Blog: 
https://fictionally-irrelevant.vercel.app/posts/be-productive-with-nemo
https://fictionally-irrelevant.vercel.app/posts/nemo-your-digital-place-to-focus

Like this pull request?  Vote for it by adding a :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ A curated list of awesome sdk's, tutorials, and other cool resources from the De
 - [Jimbru](https://github.com/athul/jimbru) - micro analytics server.
 - [Lupin](https://github.com/pomdtr/Lupin) - a Telegram bot for Logseq.
 - [Mira](https://github.com/abdelhai/mira) - a personal CRM.
+- [Nemo](https://github.com/harshitsinghai77/nemo-backend) - Nemo is your little helper and companion no matter if you need to focus, tune out other noises or if you want to have a moment of calm and relaxation. [https://nemo-landing-page.netlify.app/](https://nemo-landing-page.netlify.app/)
```
Comment (abdelhai) [inline README.md]:
```suggestion
- [Nemo](https://github.com/harshitsinghai77/nemo-backend) - a web app to help you relax by playing nature sounds.
```

**Comment by abdelhai** [comment]
hi @harshitsinghai77 thanks for the pr! Nemo looks cool but the description is pretty long. either accept my change or edit it to be shorter and more descriptive (~3-8 words)

**Comment by harshitsinghai77** [comment]
Hey @abdelhai, I didn't get your changes. Can you please write a comment on something?

**Review by abdelhai — CHANGES_REQUESTED** [review]

**Comment by abdelhai** [comment]
@harshitsinghai77 sorry, check again

**Comment by harshitsinghai77** [comment]
@abdelhai Does this look good?

**Review by abdelhai — APPROVED** [review]

**Pull request merged**